pycloudlib/lxd/tests/test_instance.py

```diff
@@ -36,12 +39,33 @@ def test_exec_with_run_command_on_xenial_machine(
             None, execute_via_ssh=False, series="xenial")
 
         instance._run_command(["test"], None)
-        expected_msg = (
-            "Many xenial images do not support executing commands"
-            " via exec due to missing kernel support: you may see"
-            " unavoidable failures.\nSee"
-            " https://github.com/canonical/pycloudlib/issues/132 for"
-            " details."
-        )
-        assert expected_msg in caplog.messages
+        assert any(self._missing_agent_msg in L for L in caplog.messages)
+        assert _m_subp.call_count == 1
```

Comment:
The leading underscore on `_m_subp` is a convention to indicate that it's unused: as we're now using it, we should remove the underscore. (This applies below too, of course.)

Reply:
Underscore dropped.